providers/atlassian/modules.go

```diff
@@ -1,24 +1,27 @@
 package atlassian
 
 import (
-	"github.com/amp-labs/connectors/common/paramsbuilder"
+	"github.com/amp-labs/connectors/common"
 )
 
-// ModuleJira is the module used for listing Jira issues.
-var ModuleJira = paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	Label:   "rest/api",
-	Version: "3",
-}
-
-// ModuleEmpty is used for proxying requests through.
-var ModuleEmpty = paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	Label:   "",
-	Version: "",
-}
+const (
+	// ModuleEmpty is used for proxying requests through.
+	ModuleEmpty common.ModuleID = ""
+	// ModuleJira is the module used for listing Jira issues.
+	ModuleJira common.ModuleID = "jira"
+)
 
 // supportedModules represents currently working and supported modules within the Atlassian connector.
 // Any added module should be appended here.
-var supportedModules = []paramsbuilder.APIModule{ // nolint: gochecknoglobals
-	ModuleEmpty,
-	ModuleJira,
+var supportedModules = common.Modules{ // nolint: gochecknoglobals
+	ModuleEmpty: {
+		ID:      ModuleEmpty,
+		Label:   "",
+		Version: "",
+	},
+	ModuleJira: {
+		ID:      ModuleJira,
```

Comment:
This ID must be declared on a module and must match the index. `ValidateParams` enforces this, so nobody would forget this.